liebs_picker.py

```diff
@@ -37,8 +37,10 @@ def INPUT_TYPES(s):
             }
         }
 
-    RETURN_TYPES = ("IMAGE",)
-    RETURN_NAMES = ("IMAGES",)
+    INPUT_IS_LIST = True
+    OUTPUT_IS_LIST = (False, True,)
+    RETURN_TYPES = ("IMAGE", "IMAGE",)
+    RETURN_NAMES = ("IMAGES", "IMAGES",)
```

Comment:
Happy to use a different name here, but this is already pretty obvious because the output link icon is different.